app/app.js

```diff
@@ -1,14 +1,23 @@
 import Ember from 'ember';
-import Resolver from 'ember/resolver';
+import Resolver from './resolver';
 import loadInitializers from 'ember/load-initializers';
+import config from './config/environment';
+
+let App;
 
 Ember.MODEL_FACTORY_INJECTIONS = true;
 
-var App = Ember.Application.extend({
-	modulePrefix: 'myapp', // TODO: loaded via config
-	Resolver: Resolver
+marked.setOptions({
+  renderer: new marked.Renderer(),
+  breaks: true
+});
+
+App = Ember.Application.extend({
+  modulePrefix: config.modulePrefix,
+  podModulePrefix: config.podModulePrefix,
+  Resolver
```

Comment:
'object short notation' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).